src/deploy.sh

```diff
@@ -574,6 +574,8 @@ function prepare_remote_dir()
     cmd_manager "$flag" "$cmd"
   fi
 
+  # Removes temporary directory if already existent
+  cmd_remotely "rm --preserve-root=all --recursive --force -- $KW_DEPLOY_TMP_FILE" "$flag"
```

Comment:
Also, always check the variable before trying to run `rm` on it since it might have the wrong path.